crates/ubrn_cli/src/android.rs

```diff
@@ -232,12 +232,12 @@ impl AndroidArgs {
             .arg(target.to_string())
             .arg("--platform")
             .arg(format!("{}", api_level));
-        if !self.common_args.release {
+        if self.common_args.profile() != "release" {
```

Comment:
Nit: extract a local variable
```suggestion
        let profile = self.common_args.profile();
        if profile != "release" {
```